sdk/tests/conformance/misc/object-deletion-behaviour.html

```diff
@@ -376,6 +376,48 @@
 shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, bufferElement)");
 shouldBeNull("gl.getParameter(gl.ELEMENT_ARRAY_BUFFER_BINDING)");
 
+if (contextVersion > 1) {
+  var bufferBaseUniform = gl.createBuffer();
+  shouldBeNonNull("bufferBaseUniform");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.bindBufferBase(gl.UNIFORM_BUFFER, 0, bufferBaseUniform)");
+  shouldBe("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)", "bufferBaseUniform");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.deleteBuffer(bufferBaseUniform)");
+  shouldBeFalse("gl.isBuffer(bufferBaseUniform)");
+  shouldBeNull("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)");
+  shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBufferBase(gl.UNIFORM_BUFFER, 0, bufferBaseUniform)");
+  shouldBeNull("gl.getParameter(gl.UNIFORM_BUFFER_BINDING)");
+
+  var bufferBaseTransformFeedback = gl.createBuffer();
+  shouldBeNonNull("bufferBaseTransformFeedback");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, bufferBaseTransformFeedback)");
+  shouldBe("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)", "bufferBaseTransformFeedback");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.deleteBuffer(bufferBaseTransformFeedback)");
+  shouldBeFalse("gl.isBuffer(bufferBaseTransformFeedback)");
+  shouldBeNull("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)");
+  shouldGenerateGLError(gl, gl.INVALID_OPERATION, "gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, bufferBaseTransformFeedback)");
+  shouldBeNull("gl.getParameter(gl.TRANSFORM_FEEDBACK_BUFFER_BINDING)");
+
+  var bufferRangeUniform = gl.createBuffer();
+  shouldBeNonNull("bufferRangeUniform");
+  shouldGenerateGLError(gl, gl.NO_ERROR, "gl.bindBufferRange(gl.UNIFORM_BUFFER, 0, bufferRangeUniform, 0, 4)");
```

Comment:
offset == 4 may not always work.  It needs to be a multiple of UNIFORM_BUFFER_OFFSET_ALIGNMENT